src/main/java/bio/terra/cli/businessobject/resource/BqResolvedOptions.java

```diff
@@ -2,16 +2,24 @@
 
 /** This enum specifies the possible ways to resolve a BigQuery resource. */
 public enum BqResolvedOptions {
-  FULL_PATH, // For data table: [project id].[dataset id].[data table id if applicable]
+  FULL_PATH, // For data table: [project id]:[dataset id].[data table id if applicable]
```

Comment:
Suggest FULL_PATH -> FULL_PATH_BQ and some comment changes

```
// project:dataset.table, used for bq CLI
FULL_PATH_BQ,
// project.dataset.table, used for SQL
FULL_PATH_SQL,
```
(I left out `if applicable` for brevity..)